packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -364,7 +363,10 @@ export default {
       this.saving = true
 
       const params = {
-        expireDate: DateTime.fromJSDate(this.expireDate).endOf('day').toISODate(),
+        expireDate: DateTime.fromJSDate(this.expireDate)
+          .setLocale(this.$language.current)
+          .endOf('day')
+          .toFormat("yyyy-MM-dd'T'HH:mm:ssZZZ"),
```

Comment:
You chose this format because Reva is not compatible with the standard ISO format, right? If so, we should create tickets in Reva to fix that and in Web to be able to track it. What do you think?

Reply:
I've created https://github.com/cs3org/reva/issues/2322. I'll create the corresponding Web issue as well once this has been acknowledged and accepted from their side.